### ISSUE TITLE
Escaped curlies cause incorrect indentation error

### ISSUE THREAD

**Issue by mriska** [rwjblue/ember-template-lint#54]
The following template:
```handlebars
<div>
  \{{example}}
</div>
```
causes the following error:
```
block-indentation: Incorrect indentation for `{{example}}
` beginning at L2:C3. Expected `{{example}}
` to be at an indentation of 2 but was found at 0.(dummy/pods/examples/computed-properties/any/template @ L2:C3): 
`<div>
  \{{example}}
</div>`
```
It seems like the indentation calculation is thrown off by the line starting with the sequence `\{{`.
